### PR TITLE
Update dependencies and documentation 

### PR DESCRIPTION
**Pull request by boeckMt**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/dlr-eoc/ukis-frontend-libraries/blob/master/CONTRIBUTING.md#-commit-message-guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] The Project is [building and passes all tests](https://github.com/dlr-eoc/ukis-frontend-libraries/blob/master/DEVELOPMENT.md#further-you-can-test-and-build-locally) without errors

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [x] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Update to Angular v13 should be made.

## What is the new behavior?

- Update Angular to v13 and other dependencies 13735d2d1aab56019787d8deaafae468589e24e0, 95d77245180894e7e3bd0142a023645d7a7cd8a7, 940e5264fc0e1692c323619d3e80161cb326aa64
- Update Node in CI for Angular update da1260d8f8c668b0e359a89eba9b81366fb30968, 99d2f8e0138c0120887c827c8a7983a1de0c63a1, 99d2f8e0138c0120887c827c8a7983a1de0c63a1, 1f4a91abff76437cdf3626136566563ae920b87d, dab8a014bec0000b4b104740e2f0883fc741f4e0
- fix some type errors in `@dlr-eoc/core-ui` schematics and add `skipLibCheck` on `ng-add` to tsconfig 55e770065b3da9ff0ac1bae17ad9fa8108d1857b, 5f146b4e53149aecfcbe61273826a3648c0d3467, 68abb5bd4a767058ccc79558694cda538177c736
- fix some type errors in `@dlr-eoc/user-info` for `AbstractControl` from `FormGroup` 83549dfe364ba4453a0582be917a43a48d593051
- fix some type errors for `OffscreenCanvas` in `demo-maps`, `demo-auth`, '@dlr-eoc/map-ol', `@dlr-eoc/map-three`, `@dlr-eoc/map-tools`  and `@dlr-eoc/utils-maps` 33be2fe6886ef88585f14d263383aed3e1c66f1d

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

## Other information
Is it part of one/more [packages](https://github.com/dlr-eoc/ukis-frontend-libraries/packages) and which?

- All `@dlr-eoc` due to Angular update

### CODE REVIEW

File: .github/workflows/package-pre-release.yml [hunk 1]
```diff
@@ -94,7 +94,7 @@ jobs:
           path: dist
       - uses: actions/setup-node@v2
         with:
-          node-version: 15
+          node-version: 16
```
Comment (MichaelLangbein) [inline .github/workflows/package-pre-release.yml:97]: Cool thing that we're moving up to node16. I think I remember that older versions of node/npm would create an old-style package-lock v1.

File: projects/core-ui/README.md [hunk 2]
```diff
@@ -78,15 +78,15 @@ main-container
     content-area
 ```
 
-and be aware that the router is placing the routes outside of the router-outlet (so they are not childs)
+and keep in mind that the router is placing the routes outside of the router-outlet (so they are not childs)
```
Comment (MichaelLangbein) [inline projects/core-ui/README.md:81]: ... `(so they are not children)`

**Review by MichaelLangbein — APPROVED** [review]
Nothing to add! Looks good to me!

**Review by lucas-angermann — APPROVED** [review]
I have nothing to add as well!

File: .github/workflows/package-pre-release.yml (hunk 1 above)
Comment (boeckMt) [reply .github/workflows/package-pre-release.yml:97]: Yes I think this was between [npm 6 and 7](https://github.blog/2021-02-02-npm-7-is-now-generally-available/#changes-to-the-lockfile). 
We only have one Issue to fix for the publish step #129 where we use npm 6. But we don't install anything here, it's just for the publish command so I hope this is still working with the newer node version.

**Pull request merged**